src/Keboola/StorageApi/Client.php

```diff
@@ -756,14 +756,25 @@ public function writeTableAsyncDirect($tableId, $options = array())
     /**
      * @param $tableId
      * @param array $options
-     * @return mixed|string
+     * @return int
```

Comment:
porad je to string a ani do budoucna bych se nesnazil tvrdit, ze neni

Reply:
![image](https://user-images.githubusercontent.com/497675/50095598-2e401b00-0216-11e9-867b-d961f5cd356c.png)

api vrací id jobu jako integer a to probublá i dál

![image](https://user-images.githubusercontent.com/497675/50095772-824aff80-0216-11e9-8aca-c96e0a3ee16f.png)

Reply:
och, no tak to bysme meli zmenit dokud nam to nepretika

Reply:
ještě chvíli to vydrží 

- https://www.elastic.co/guide/en/elasticsearch/reference/2.3/number.html
- https://keboola.docs.apiary.io/#reference/tickets/generate-unique-id/generate-new-id
- https://github.com/keboola/connection/blob/f9204c02715e43eb91f9b182fc48d172cc3bb37a/sql/migrations/accounts/Version20130122163430.php#L19

Reply:
hmm, jenže tohle jsou sapi joby ne eventy a ty jsou v mysql https://github.com/keboola/connection/blob/194f3a8a4edcf2562a1dec579e9e81c8116dbc53/sql/migrations/accounts/Version20130528154408.php#L17

Reply:
a id jobu je číslo aby se dalo řadit

Reply:
no slo mi o jinou vec, kdyz se udela:

```
<?php

$json = '{"a": 9223372036854775808}';
$ret = json_decode($json);
var_dump($ret);
```

tak to vrati
```
class stdClass#1 (1) {
  public $a =>
  double(9.2233720368548E+18)
}
```

bez jakekoliv chyby a to je dost pruser. Z principu bysme teda to ID nemeli povazovat za cislo - protoze vyznam cisla nema - a radit se da i podle stringu.
Tohle nema nic spolecnyho s interni reprezentaci v DB, ale s tim jak se to prezentuje na strane API. 

(nastesti jsme od tohohle problemu jeste docela daleko )